### PR TITLE
Internal setters for Conflicts

### PR DESCRIPTION
**Pull request by ealsur**
Correcting setter accessor for `CosmosConflicts`

### CODE REVIEW

File: Microsoft.Azure.Cosmos/src/Resource/Settings/CosmosAccountSettings.cs [hunk 1]
```diff
@@ -71,7 +71,7 @@ public virtual IEnumerable<CosmosAccountLocation> ReadableLocations
         /// </para>
         /// </remarks>
         [JsonProperty(PropertyName = Constants.Properties.Id)]
-        public virtual string Id { get; set; }
+        public virtual string Id { get; internal set; }
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos/src/Resource/Settings/CosmosAccountSettings.cs]: mark 'ResourceId ' internal.

Comment (kirankumarkolli) [reply Microsoft.Azure.Cosmos/src/Resource/Settings/CosmosAccountSettings.cs]: Also please move all pubic to the top.

File: Microsoft.Azure.Cosmos/src/Resource/Conflict/CosmosConflict.cs [hunk 2]
```diff
@@ -34,14 +34,14 @@ public class CosmosConflict
         /// </para>
         /// </remarks>
         [JsonProperty(PropertyName = Documents.Constants.Properties.Id)]
-        public virtual string Id { get; set; }
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos/src/Resource/Conflict/CosmosConflict.cs]: Lets rename 'CosmosConflict' To 'CosmosConflictSettings'.
I might have said it other way earlier :-)

File: Microsoft.Azure.Cosmos/src/Resource/Settings/CosmosAccountSettings.cs (hunk 1 above)
Comment (ealsur) [reply Microsoft.Azure.Cosmos/src/Resource/Settings/CosmosAccountSettings.cs]: Done

File: Microsoft.Azure.Cosmos/src/Resource/Conflict/CosmosConflict.cs (hunk 2 above)
Comment (ealsur) [reply Microsoft.Azure.Cosmos/src/Resource/Conflict/CosmosConflict.cs]: Done

File: Microsoft.Azure.Cosmos/src/Resource/Settings/CosmosAccountSettings.cs [hunk 3]
```diff
@@ -71,33 +71,33 @@ public virtual IEnumerable<CosmosAccountLocation> ReadableLocations
...
         /// <remarks>
-        /// A Resource Id is the unique, immutable, identifier assigned to each Azure Cosmos DB 
-        /// resource whether that is a database, a collection or a document.
-        /// These resource ids are used when building up SelfLinks, a static addressable Uri for each resource within a database account.
+        /// ETags are used for concurrency checking when updating resources. 
         /// </remarks>
-        [JsonProperty(PropertyName = Constants.Properties.RId)]
-        public virtual string ResourceId { get; protected internal set; }
+        [JsonProperty(PropertyName = Constants.Properties.ETag)]
+        public virtual string ETag { get; protected internal set; }
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos/src/Resource/Settings/CosmosAccountSettings.cs]: Lets just make it "Internal" only.

Comment (ealsur) [reply Microsoft.Azure.Cosmos/src/Resource/Settings/CosmosAccountSettings.cs]: Done

File: Microsoft.Azure.Cosmos/src/Resource/Settings/CosmosAccountSettings.cs [hunk 4]
```diff
@@ -71,33 +71,33 @@ public virtual IEnumerable<CosmosAccountLocation> ReadableLocations
...
+        /// Gets or sets the Resource Id associated with the resource in the Azure Cosmos DB service.
         /// </summary>
         /// <value>
-        /// The entity tag associated with the resource.
+        /// The Resource Id associated with the resource.
         /// </value>
         /// <remarks>
-        /// ETags are used for concurrency checking when updating resources. 
+        /// A Resource Id is the unique, immutable, identifier assigned to each Azure Cosmos DB 
+        /// resource whether that is a database, a collection or a document.
```
Comment (simplynaveen20) [inline Microsoft.Azure.Cosmos/src/Resource/Settings/CosmosAccountSettings.cs]: Want to change to container and item instead of collection and document ?

Comment (ealsur) [reply Microsoft.Azure.Cosmos/src/Resource/Settings/CosmosAccountSettings.cs]: It's an internal property, the PR shows that I changed the text but I just reordered based on code guidelines

Comment (ealsur) [reply Microsoft.Azure.Cosmos/src/Resource/Settings/CosmosAccountSettings.cs]: If it was a public property, updating the text would make sense 🙂

**Pull request merged**